modules/metastorage-server/src/main/java/org/apache/ignite/internal/metastorage/server/raft/MetaStorageListener.java

```diff
@@ -74,7 +76,7 @@
     private final KeyValueStorage storage;
 
     /** Cursors map. */
-    private final Map<IgniteUuid, IgniteBiTuple<Cursor<?>, CursorType>> cursors;
+    private final Map<IgniteUuid, IgniteBiTuple<Cursor<?>, CursorMeta>> cursors;
```

Comment:
It looks complicated. If I were you, I would add Cursor to CursorMeta.

Reply:
Fixed.